### PR TITLE
DNM - testing zuul with vyos node

### PR DESCRIPTION
**Pull request by NilashishC**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/30

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

File: tests/foo.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      shell: ansible-galaxy install ansible-network.network-engine
+
+    - name: Set Fact
+      set_fact:
+        ansible_network_os: vyos
+
+    - name: run ansible
+      args:
+        chdir: ~/src/github.com/ansible-network/vyos
+      shell: ansible-playbook -i /etc/ansible/hosts tests/test_vyos_interfaces.yaml -e "ansible_network_os=vyos ansible_connection=network_cli"
```
Comment (pabelanger) [inline tests/foo.yaml]: ansible-playbook -c network_cli tests/test_vyos_interfaces.yaml -e "ansible_password=vyos"

**Comment by pabelanger** [comment]
recheck

File: tests/foo.yaml (hunk 1 above)
Comment (NilashishC) [reply tests/foo.yaml]: Fixed. Now running `ansible-playbook -c network_cli tests/test_vyos_interfaces.yaml -e "ansible_user=vyos ansible_password=vyos"`

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

File: module_utils/vyos/facts/interfaces/interfaces.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+The vyos interfaces fact class
+It is in this file the configuration is collected from the device
+for a given resource, parsed, and the facts tree is populated
+based on the configuration.
+"""
+from copy import deepcopy
+from ansible.module_utils. \
+     vyos.facts.base import FactsBase
+from re import findall, M
+import q
```
Comment (pabelanger) [inline module_utils/vyos/facts/interfaces/interfaces.py]: delete this line, and test should pass. We'll make sure to bootstrap the project properly moving forward

Comment (NilashishC) [reply module_utils/vyos/facts/interfaces/interfaces.py]: Fixed

**Comment by pabelanger** [comment]
recheck

**Pull request closed without merging**